src/containers/pages/InterventionPlan/UpdatePlan/index.tsx

```diff
@@ -54,25 +57,23 @@ const UpdatePlan = (props: RouteComponentProps<RouteParams> & UpdatePlanProps) =
   /** async function to load the data */
   async function loadData() {
     try {
-      setLoading(plan === null); // only set loading when there are no plans
       const planFromAPI = await apiService.read(planIdentifier);
       fetchPlan(planFromAPI);
     } catch (e) {
-      // do something with the error?
-    } finally {
-      setLoading(false);
+      growl(e.message, { type: toast.TYPE.ERROR });
     }
   }
 
   useEffect(() => {
     loadData();
+    return () => controller.abort();
   }, []);
 
-  if (loading === true) {
+  if (plan === null) {
     return <Loading />;
   }
```

Comment:
Same as above, may need a message like "Plan not found" or something

Reply:
Created #798, to help guid discussions and work for this